src/components/editor-page/editor-page.tsx

```diff
@@ -8,7 +8,6 @@ import React, { Fragment, useCallback, useEffect, useRef, useState } from 'react
 import { useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
 import { useLocation } from 'react-router'
-import useMedia from 'use-media'
```

Comment:
Do we use the package anywhere else? If not we should remove it.

Reply:
Good point. I removed it